addon/index.js

```diff
@@ -45,13 +47,39 @@ const button = ActionButton({ // eslint-disable-line new-cap
     '32': './icon-32.png',
     '64': './icon-64.png'
   },
-  badge: state.updates.length,
-  badgeColor: prefs.BADGE_COLOR,
   onClick: function() {
     tabs.open({ url: prefs.CONTENT_URL });
   }
 });
 
+function getInstalledAddons(cb) {
+  request({
+    url: 'http://ideatown.dev:8000/api/experiments?format=json',
+    onComplete: res => {
+      if (!res.json.results.length || res.error) {
+        console.error('No experiments:: ', res);
+      }
+
+      store.availableExperiments = res.json.results.map(exp => {
+        return exp.addon_id;
+      });
+
+      AddonManager.getAllAddons(addons => {
+        addons.forEach(ex => {
+          if (isIdeatownAddon) {
+            store.experiments[ex.id] = ex.version;
+          }
+        });
+
+        // send back to client as an array
+        cb(Object.keys(store.experiments).map(k => {
+          return {id: k, version: store.experiments[k]};
+        }));
```

Comment:
OK, so you have an object of the form
```
store.experiments = { id: version, id: version }
```

and you want to convert it into an array of objects of the form 

```
{id: 'some-addon-id', version: '1.0.1'}
```

One nice way to do this might be
```
let output = [];
for (ex in experiments) {
  output.push({
    id: ex,
    version: experiments[ex]
  });
}
return output;
```

Seems like the `for in` version is more readable. Thoughts?